specs/waku/mailserver.md

```diff
@@ -69,7 +69,7 @@ If `Topics` is used the `Cursor` field MUST be specified for the argument order
 
 ### Receiving Historic Envelopes
 
-Historic envelopes MUST be sent to a peer as a packet with a P2P Message code (`0x7f`) followed by an array of Waku envelopes.
+Historic envelopes MUST be sent to a peer as a packet with a P2P Message code (`0x7f`) followed by an array of Waku envelopes. A Mailserver MUST limit the amount of messages sent, either by the `Limit` specified in the request or limited to the maximum [RLPx packet size](./waku#maximum-packet-size), whichever limit comes first.
```

Comment:
Is this a MUST or a SHOULD? Does this MUST have any implications on backwards compatibility? What happens if a mailserver misbehaves?

Reply:
So I believe the mailservers already do, right @cammellos? And even if it is a MUST, it doesn't have any effects on the client other than dropping messages if they are too large. Additionally if they don't do it a peer could just request smaller sizes again by requesting less messages.

I don't think the must has many implications on backward compatibility cause I can just request smaller chunks and pagination is already a thing.

Reply:
it's already as described, we batch based on size (or we include all the envelopes if limit is reached first)

Reply:
Ok so MUST seems fine. Are we specifying that clients should drop messages that are too large?

Reply:
@oskarth yup, check the waku spec changes. Already there.